src/main/java/com/google/step/datamanager/DealManagerDatastore.java

```diff
@@ -141,21 +145,42 @@ public Deal updateDeal(Deal deal, List<String> tagNames) {
     return transformEntityToDeal(dealEntity);
   }
 
+  /** Retrieves deals posted by restaurants or users */
+  private List<Deal> getDealsPublishedByRestaurantsOrUsers(
+      Set<Long> idsOfFollowedFieldName, String filterAttribute) {
```

Comment:
```suggestion
      Set<Long> ids, String filterAttribute) {
```

Don't need a long name here, the method name provides the context.